tests/PluginLoaderTest.php

```diff
@@ -44,6 +45,27 @@ public function testReturnsPlugins(): void
         $this->assertInstanceOf(FooBarPlugin::class, $pluginLoader->getInstances()['foo/bar-bundle']);
     }
 
+    public function testSkipsDisabledPlugins(): void
+    {
+        include_once __DIR__.'/Fixtures/PluginLoader/FooBarPlugin.php';
+        include_once __DIR__.'/Fixtures/PluginLoader/FooDisabledPlugin.php';
+
+        $pluginLoader = new PluginLoader(
+            null,
+            [
+                'foo/bar-bundle' => new FooBarPlugin(),
+                'foo/enabled-bundle' => new FooDisabledPlugin(false),
+                'foo/disabled-bundle' => new FooDisabledPlugin(true),
+            ]
+        );
+
+        $this->assertArrayHasKey('foo/bar-bundle', $pluginLoader->getInstances());
+        $this->assertArrayHasKey('foo/enabled-bundle', $pluginLoader->getInstances());
+        $this->assertArrayHasKey('foo/disabled-bundle', $pluginLoader->getInstances());
```

Comment:
shouldn’t this be the other way around?
```suggestion
        $this->assertArrayNotHasKey('foo/disabled-bundle', $pluginLoader->getInstances());
```

Reply:
fixed in 3f559bc235ea2288d1b5ae3f00bbe52781427fdf